src/main.js

```diff
@@ -41,6 +37,24 @@ function toggleDirectory(nameElement) {
     nameElement.parentNode.classList.add("fs-api-directory-collapse");
   }
 }
+
+/**Selects the clicked element and unselects any other that might be selected
+ * @param {HTMLLIElement} liElement - The item that is going to be selected*/
+function selectItem(liElement) {
```

Comment:
1. We will have to restrict selected items in our current tree (for some reason we might have multiple `fs-api-js` in our page
2. We can simplify the iteration by selecting using the  `fs-api-selected` class
3. It's better to rename this to `selectEntry` since we call them "entries"

Also, I would consider extracting the deselection of entries into its own function.

## Example

```js
function selectEntry(root, entry) {
  Array.from(root.querySelectorAll('.fs-api-selected')).forEach(entry => entry.classList.remove('fs-api-selected'));
}
```